go.mod

```diff
@@ -1,43 +1,44 @@
 module github.com/cockroachdb/pebble
 
 require (
-	github.com/DataDog/zstd v1.4.5
+	github.com/DataDog/zstd v1.5.2
```

Comment:
I don't think we want all of these version bumps in the same PR. Those would ideally be handled separately. For some of the dependencies (`xxhash`, `compress`, etc.), we'll also want to run some benchmarks. There have been some issues in the past.

Can we keep this PR focused on bumping the build version to `1.19` (i.e. should be a single line change)?

Reply:
Yes, we can keep this PR focused on bumping the build version to 1.19. I'll create separate PRs for the other version bumps and will make sure to run the necessary benchmarks before submitting them. Thanks for the suggestion.

Reply:
@nicktrav you can check this PR again, I bumped only the build version to `1.19`. Other than that, I have run the Makefile locally and everything looks fine.